about.html

```diff
@@ -4,13 +4,28 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="./mobileMenu.js"></script>
   <link rel="stylesheet" href="./about.css">
+  <link rel="stylesheet" href="./mobileMenu.css">
   <title>About E3 2023</title>
 </head>
 <body class="body">
 
+  <div class="mobileMenu">
```

Comment:
- [ ] Good job but your about page design is a bit different from the original design :cry: You can check the difference with the screenshots I'm sharing with you below :arrow_down:  Please make sure to fix it in order to make your design work as expected :+1: 

#### Your design
![about21](https://user-images.githubusercontent.com/6615394/210892219-3118a16f-bf92-4f45-ba65-f148e0b0ed7b.jpg)
![about11](https://user-images.githubusercontent.com/6615394/210892221-493bce7c-feb2-4539-8630-42b1f5f5ef2b.jpg)

#### Original design
![about_design](https://user-images.githubusercontent.com/6615394/210892739-9a698a5d-36f7-4363-a666-7890975a02c3.jpg)